samples/wifi/shell/README.rst

```diff
@@ -51,12 +51,25 @@ The following is an example of the CLI command:
 
    west build -b nrf5340dk_nrf5340_cpuapp -- -DSHIELD=nrf7002ek
 
-To build for the nRF9160 DK, use the ``nrf9160dk_nrf9160_ns`` build target with the ``SHIELD`` CMake option set to ``nrf7002ek`` and scan-only overlay configuration.
-The following is an example of the CLI command:
+.. tabs::
 
-.. code-block:: console
+   .. group-tab:: nRF9161 DK
+
+      To build for the nRF9161 DK, use the ``nrf9161dk_nrf9161_ns`` build target with the ``SHIELD`` CMake option set to ``nrf7002ek`` and a scan-only overlay configuration.
+      The following is an example of the CLI command:
+
+      .. code-block:: console
+
+         west build -p -b nrf9161dk_nrf9161_ns -- -DOVERLAY_CONFIG=overlay-scan-only.conf -DSHIELD=nrf7002ek
```

Comment:
We could not use `nrf7002ek_nrf7000` itself, but can be done in another PR as well.